sdlf-cicd/nested-stacks/template-cicd-cfn-module.yaml

```diff
@@ -244,8 +250,13 @@ Resources:
       EncryptionKey: !Ref pKMSKey
       VpcConfig: !If
         - RunInVpc
-        - SecurityGroupIds: !Split [",", !ImportValue sdlf-cicd-prerequisites-vpc-security-groups]
-          Subnets: !Split [",", !ImportValue sdlf-cicd-prerequisites-vpc-subnets]
```

Comment:
same here, weird formatting change... not a big deal though.

Reply:
left as it is